src/webfaf/templates/base.html

```diff
@@ -18,7 +18,6 @@
       ROOT_URL = "{{url_for('root')}}";
     </script>
     <script src="{{ url_for('system_static', component='jquery', filename='latest/jquery.min.js')}}"></script>
-    <script src="{{ url_for('system_static', component='datatables', filename='js/jquery.dataTables.min.js')}}"></script>
```

Comment:
If you remove this, you can also remove the `xstatic-datatables-common` from the spec file.